docs/en/latest/terminology/plugin.md

```diff
@@ -80,10 +80,25 @@ Some common configurations can be applied to plugins through the `_meta` configu
 
 | Name         | Type | Description |
 |--------------|------|-------------|
+| disable      | boolean  | Whether to disable the plugin |
 | error_response | string/object  | Custom error response |
 | priority       | integer        | Custom plugin priority |
 | filter  | array | Depending on the requested parameters, it is decided at runtime whether the plugin should be executed. Something like this: `{{var, operator, val}, {var, operator, val}, ...}}`. For example: `{"arg_version", "==", "v2"}`, indicating that the current request parameter `version` is `v2`. The variables here are consistent with NGINX internal variables. For details on supported operators, please see [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list). |
 
+### Disable the plugin
+
+Through the `disable` configuration, you can add a new plugin with disabled status and the request will not go through the plugin. This feature is used in the APISIX Dashboard.
```

Comment:
We don't need to mention the Dashboard as this feature can be used by any other components too.

Reply:
done